src/Engines/WebGPU/Extensions/engine.dynamicTexture.ts

```diff
@@ -16,10 +16,19 @@ WebGPUEngine.prototype.updateDynamicTexture = function(texture: Nullable<Interna
         gpuTextureWrapper = this._textureHelper.createGPUTextureForInternalTexture(texture, width, height);
     }
 
-    this._textureHelper.updateTexture(canvas, texture, width, height, texture.depth, gpuTextureWrapper.format, 0, 0, invertY, premulAlpha, 0, 0, this._uploadEncoder);
+    this.createImageBitmap(canvas).then((bitmap) => {
+        this._textureHelper.updateTexture(bitmap, gpuTextureWrapper.underlyingResource!, width, height, texture.depth, gpuTextureWrapper.format, 0, 0, invertY, premulAlpha, 0, 0, this._uploadEncoder);
+        if (texture.generateMipMaps) {
+            this._generateMipmaps(texture, this._uploadEncoder);
+        }
+
+        texture.isReady = true;
+    });
+
+    /*this._textureHelper.updateTexture(canvas, texture, width, height, texture.depth, gpuTextureWrapper.format, 0, 0, invertY, premulAlpha, 0, 0, this._uploadEncoder);

```

Comment:
Only one question, I wonder why keeping the comment ???

Reply:
Because as soon as `copyExternalImageToTexture` is fixed in Chrome I will remove the code above line 27 and will uncomment the code starting at line 28 :)